src/frontend/src/btc/stores/utxos-fee.store.ts

```diff
@@ -0,0 +1,29 @@
+import type { UtxosFee } from '$btc/types/btc-send';
+import type { Option } from '$lib/types/utils';
+import { writable, type Readable } from 'svelte/store';
+
+export type UtxosFeeStoreData = Option<{
+	utxosFee?: UtxosFee;
+}>;
+
+export interface UtxosFeeStore extends Readable<UtxosFeeStoreData> {
+	setUtxosFee: (data: UtxosFeeStoreData) => void;
+}
+
+export const initUtxosFeeStore = (): UtxosFeeStore => {
```

Comment:
Can you provide dedicated tests for the store?

Reply:
I created initially a few tests there, now added additional test for `reset`. Please let me know if you have some other cases in mind.